Modules/Loadable/Markups/MRML/vtkMRMLMarkupsAngleNode.h

```diff
@@ -43,6 +43,14 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsAngleNode : public vt
 
   const char* GetIcon() override {return ":/Icons/MarkupsAngleMouseModePlace.png";}
 
+  enum AngleModes
```

Comment:
Could you document the enum ?   Adding something like this would work

```
\sa SetAngleModeToUnsigned()
\sa SetAngleModeToOrientedSigned()
\sa SetAngleModeToOrientedPositive()
```